app/data/mysql/account_store_test.go

```diff
@@ -11,10 +13,37 @@ import (
 func TestAccountStore(t *testing.T) {
 	db, err := mysql.TestDB()
 	require.NoError(t, err)
-	store := &mysql.AccountStore{db}
+	var store data.AccountStore = &mysql.AccountStore{db}
```

Comment:
Not sure I understand the need for declaration change here.

Reply:
Due to the issue below, previously store wasn't an interface.

![Screenshot 2024-04-17 at 19 54 06](https://github.com/keratin/authn-server/assets/2347485/fc49dafc-ee87-4866-b424-218d10c1c4e5)

Reply:
I have reverted that change now